projects/wmf_default.yaml

```diff
@@ -114,17 +114,26 @@ paths:
                 - path: sys/table.js
                   options:
                     conf: '{{options.table}}'
-            /table-ng: &sys_table_ng
+            /table3: &sys_table_ng
               x-modules:
                 - path: sys/table.js
                   options:
                     conf: '{{options.table_ng}}'
             /parsoid_bucket:
               x-modules:
-                - path: sys/parsoid_bucket.js
+                - path: sys/multi_content_bucket.js
                   options:
                     time_to_live: '{{default(options.parsoid.time_to_live, 84600)}}'
                     delete_probability: '{{default(options.parsoid.delete_probability, 1)}}'
+                    table_name_prefix: parsoid
+                    main_content_type:
```

Comment:
Complete bike-shedding, feel free to ignore, but how about s/main/primary/?

Reply:
> Complete bike-shedding, feel free to ignore, but how about s/main/primary/?

Or not...given how pervasive the term is throughout the code; Perhaps not worth the effort.